heron/statemgrs/src/java/com/twitter/heron/statemgr/FileSystemStateManager.java

```diff
@@ -161,4 +180,52 @@ public void initialize(Config config) {
   private static String concatPath(String basePath, String appendPath) {
     return String.format("%s/%s", basePath, appendPath);
   }
+
```

Comment:
Are you referring to the `doMain` functionality? That is integrated into both the local and zookeeper subclasses of `FileSystemStateManager`, so it is available in all of our state managers.